src/main/kotlin/uk/gov/gdx/datashare/repository/PublisherSubscriptionRepository.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.gdx.datashare.repository
+
+import org.springframework.data.r2dbc.repository.Query
+import org.springframework.data.repository.kotlin.CoroutineCrudRepository
+import org.springframework.stereotype.Repository
+
+@Repository
+interface PublisherSubscriptionRepository : CoroutineCrudRepository<PublisherSubscription, Long> {
```

Comment:
PublisherSubscription feels slightly awkward naming wise, but I don't have much better - I guess it's probably more `EventSubscription`, because events could conceivably come from multiple publishers?

Reply:
Done